rel-eng/tito.props

```diff
@@ -462,6 +441,28 @@ whitelist = ansiblerole-foreman_scap_client
   rubygem-smart_proxy_remote_execution_ssh
   rubygem-smart_proxy_salt
   rubygem-smart_proxy_spacewalk
+  rubygem-systemu
+  rubygem-timeliness
+  rubygem-ttfunk
+  rubygem-vault
+  rubygem-wicked
+  rubygem-zscheduler
+
+[foreman-plugins-nightly-nonscl-rhel7]
+disttag = .el7
+whitelist = ansiblerole-foreman_scap_client
+  ansiblerole-insights-client
+  foreman-discovery-image-service
+  puppet-foreman_scap_client
+  rubygem-faraday
+  rubygem-fast_gettext
+  rubygem-mysql2
+  rubygem-netrc
+  rubygem-net-ssh
+  rubygem-newt
```

Comment:
FDI.

Reply:
I'm not sure if these are supposed to be SCL'd ? https://github.com/theforeman/foreman-installer/pull/441#discussion_r364246612

Reply:
wrong link? but yeah, you had a separate PR for the FDI things.
my comment was "this is used by FDI, this is fine", I should have been more verbose.

Reply:
Yea, correct link -- https://github.com/theforeman/foreman-discovery-image/pull/120#issuecomment-571618910